sale_cancel_reason/view/sale_order_cancel_reason_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_sale_order_cancel_reason_form">
+        <field name="name">Sale Order Cancel Reason</field>
+        <field name="model">sale.order.cancel.reason</field>
+        <field name="arch" type="xml">
+            <form string="Sale Order Cancel Reasons">
+                <field name="name"/>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_sale_order_cancel_reason_tree">
+        <field name="name">Sale Order Cancel Reason</field>
+        <field name="model">sale.order.cancel.reason</field>
+        <field name="arch" type="xml">
+            <tree string="Sale Order Cancel Reasons">
+                <field name="name"/>
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="action_sale_order_cancel_reason">
+        <field name="name">Sale Order Cancel Reason</field>
+        <field name="res_model">sale.order.cancel.reason</field>
+        <field name="view_type">form</field>
```

Comment:
It seems view_type field has been removed. Please remove it from all the actions.